content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+========================================
+Create opportunities from a contact form
+========================================
+
+Adding a contact form to a website makes it easy to convert visitors into leads and opportunities.
+After a visitor submits their information, an opportunity can be created automatically, and assigned
+to a designated sales team.
+
+.. _crm/customize-contact-form:
+
+Customize contact forms
+=======================
+
+By default, the *Contact Us* page on an Odoo website displays a preconfigured contact form. This
+form can be customized, as needed, with additional fields and settings to suit the needs of a
+specific sales team.
+
+Navigate to :menuselection:`Website app --> Contact Us`, then click :guilabel:`Edit` in the
+top-right of the screen to open the web editor. Click on the form to open the form configuration
+settings. The following options are available to customize the contact form:
+
+.. image:: opportunities_form/form-customization.png
+   :align: center
+   :alt: The form configuration settings on an Odoo website.
+
+- :guilabel:`Action`: the default action for a contact form is :guilabel:`Send an Email`. Select
+  :guilabel:`Create an Opportunity` from the drop-down list to capture the information in the *CRM*.
+- :guilabel:`Sales Team`: choose sales team from the drop-down menu the opportunities from this form
+  should be assigned to.
+- :guilabel:`Salesperson`: if the opportunities should be assigned to a specific salesperson, select
+  them from the drop-down menu. If no selection is made in this field, the opportunities are
+  assigned based on the team's existing rules.
+- :guilabel:`Marked Fields`: use this field to alter how the form handles marked fields. The default
```

Comment:
What is a "marked field"? Is that a required field? How do you make one required but one not? I might just link to a doc that explains how to do that, or add a little bit more info. I wasn't sure if it meant ALL the fields, etc.